app/scripts/sync-environment-to-convex.js

```
@@ -1,85 +1,109 @@
-#!/usr/bin/env node
-
-import fs from "fs";
-import path from "path";
-import { execSync } from "child_process";
+/* eslint-disable perfectionist/sort-objects */
```

ESLint

[error] Definition for rule 'perfectionist/sort-objects' was not found.

(perfectionist/sort-objects)